src/app/components/wait/index.html

```diff
@@ -51,6 +51,12 @@
     >
```

Comment:
Should we also update the wait service methods to include the public `dispose` method at https://github.com/blackbaud/skyux-indicators/blob/master/src/app/public/modules/wait/wait.service.ts#L64? I assume this is a question for @Blackbaud-SteveBrush.

Reply:
Good catch, but I think we're okay to not include it in the docs. It's primarily used internally.

Reply:
OK. Thanks!